### PR TITLE
IRIS-849-csharp-fullpage

### PR DESCRIPTION
**Pull request by paweltomaszewskisaucelabs**
### Description
Add c# fullpage screenshot docs

### Types of Changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/visual-testing/integrations/csharp.md
Comment (FriggaHel) [inline docs/visual-testing/integrations/csharp.md:341]: Not sure that's ideal:
- Casing is not the same
- Method names are not identical (`SauceVisualCheck` vs. `VisualCheck`)
- Not mentioning the `FullPageConfig` kind.

My first feeling when reading is that example and description does not match (which is the case actually).

Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/integrations/csharp.md]: fixed

File: docs/visual-testing/integrations/csharp.md
Comment (konraddysput) [inline docs/visual-testing/integrations/csharp.md:350]: What is the default value?

Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/integrations/csharp.md:350]: there is no default value so no value == no delay

Comment (FriggaHel) [reply docs/visual-testing/integrations/csharp.md:350]: Should we mention the default behaviour ?

Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/integrations/csharp.md:350]: IMO no need.
I would expect it to be 0 if nothing was mention in the docs. What about you guys?

File: docs/visual-testing/_partials/_fullpage-description.md
Comment (konraddysput) [inline docs/visual-testing/_partials/_fullpage-description.md:9]:
```suggestion
- `delayAfterScrollMs`: Delay in ms after scrolling and before taking screenshots. The default value is 0. We recommend using this option for lazy loading content.
```

File: docs/visual-testing/integrations/csharp.md
Comment (konraddysput) [inline docs/visual-testing/integrations/csharp.md:337]: There is no way to enable this option for every VisualCheck via Client options?

**Review by konraddysput — APPROVED** [review]
Can we please wait for Felix approval to land it?

File: docs/visual-testing/integrations/csharp.md
Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/integrations/csharp.md:337]: no but we have task to do it
https://saucedev.atlassian.net/browse/IRIS-892

File: docs/visual-testing/integrations/java.md
Comment (FriggaHel) [inline docs/visual-testing/integrations/java.md:338]: Is this correct for java ? Isn't it expected to use Builder ?

Casing looks off too.

File: docs/visual-testing/integrations/java.md
Comment (FriggaHel) [inline docs/visual-testing/integrations/java.md:344]: Same as previous comment.

File: docs/visual-testing/integrations/java.md
Comment (FriggaHel) [inline docs/visual-testing/integrations/java.md:1]: Is there a need for changes to that file ? Seems like `FullPageDescription` was doing exactly this.

Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/integrations/java.md:1]: If we want to provide specific description for java then yes we need to change it

File: docs/visual-testing/integrations/java.md
Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/integrations/java.md]: fixed

File: docs/visual-testing/integrations/java.md
Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/integrations/java.md]: fixed

**Review by FriggaHel — APPROVED** [review]

**Pull request merged**
